### PR TITLE
Fix phi.js function

### LINKED ISSUE #408: Fix phi.js function
There is currently a method called `phi` that lives in `/lib/phi.js`.
It is incomplete and needs to be fixed!

### Acceptance Criteria:
1. Running `yarn test phi` results in tests passing.
2. You have written a skeleton method for someone else to work on.
3. You have written tests surrounding your skeleton method.
4. Running `yarn lint` does not print any errors to the console!
5. Optional: write one or two more tests surrounding the method you are solving to account for potential edge cases.

### Please include the skeleton of a new method + an accompanying test for someone else to work on at the time of creating a pull request! A pull request will most likely be denied if it does not contain a skeleton method for someone else to work on! For more information, please read the [Contributing Guide](https://github.com/BlakeGuilloud/ganon/blob/master/CONTRIBUTING.md).

Thank you so much for your contribution!

### PR DESCRIPTION
**Pull request by Yogendra0Sharma**
Closes #408 

- [X] Running `yarn lint` does not trigger any linter errors
- [X] The test of the method you have fixed is passing
- [X] You have written a new skeleton method for someone else to work on!
- [X] You have written tests to accompany your skeleton method

### CODE REVIEW

File: lib/convertKilometerToMach.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const { convertKilometerToMach } = require("../lib");
```
Comment (ktilcu) [inline lib/convertKilometerToMach.test.js]: I think this file is named improperly. shouldn't have the `test` in it. It should also not be the tests ;)

File: test/convertKilometerToMach.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const { convertKilometerToMach } = require("../lib");
+
+describe("convertKilometerToMach", () => {
+  test("1 kilometer is 0.000809848 march", () => {
+    expect(convertKilometerToMach(1)).toBeCloseTo(0.000809848, 4);
+    expect(convertKilometerToMach(5.5)).toBeCloseTo(0.00445416, 4);
```
Comment (ktilcu) [inline test/convertKilometerToMach.test.js]: Lets either generalize the `test` description or move each `expect` into it's own `test`block.

**Comment by ktilcu** [comment]
@Yogendra0Sharma We really appreciate the contributions. If we could get this wrapped up it would be great. Thanks!
